### PR TITLE
fix crds

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Towards: giantswarm/giantswarm#14076

### CODE REVIEW

File: helm/prometheus-operator-app/templates/crd-install/crd-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{{/*
+We have to create individual configmaps for each CRD - they exceed the total
+allowed length for a configmap if they are combined.
+*/}}
+{{- if .Values.crds.install }}
+{{- range $path, $bytes := $.Files.Glob "files/**" }}
```
Comment (glitchcrab) [inline helm/prometheus-operator-app/templates/crd-install/crd-configmap.yaml:6]: very sexy

**Review by glitchcrab — APPROVED** [review]
looks good to me, i will be stealing the way you've ranged over the crd files for us in the cert-manager chart :D

**Comment by TheoBrigitte** [comment]
> looks good to me, i will be stealing the way you've ranged over the crd files for us in the cert-manager chart :D

please do :)

**Comment by TheoBrigitte** [comment]
It works :gift: :christmas_tree: :santa: 

1. crd-install job ran nicely
image
2. CRDs are installed
image
3. prometheus-operator does reconcile and is not complaining about missing resource types
image

**Pull request merged**